### DISCUSSION TITLE
Some activities should un/block speak.

### DISCUSSION THREAD

**Discussion by dDeepLb** [littlesera/LSCG#203] [Ideas]
That should be high priority, as many players want it. These can be new or existing actions while we don't have text commands. You can then add commands or replace actions with them.

**Comment by littlesera** [comment]
Could you elaborate?

**Comment by dDeepLb** [comment]
When you hypnotised you can't talk. We want to be able to allow someone to speak or block if allowed. Now we can boop someone to unzonk. You can make activity to allow/block someone to speak, as for now we don't have text commands, as was said in #17 . When text commands will be implemented, you can add them as addition or just replace above mentioned activities.
That can be just temporary replacement for allowing/blocking someone something. (Also be good addition to allow/block to change and access someones inventories.)

**Comment by littlesera** [comment]
I see, so specifically while someone is hypnotized, allowing them to speak to allow "repeat after me" kind of RP?
